### PR TITLE
add username

### PR DESCRIPTION
**Pull request by marcosberges**

### CODE REVIEW

**Comment by daguito81** [comment]
Hey marcos, could you move the file into the names folder? You cna just move it locally and then do git add, commit then push

**Comment by marcosberges** [comment]
> Hey marcos, could you move the file into the names folder? You cna just move it locally and then do git add, commit then push

Done! thanks for pointing it out.

File: marcosberges.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+name: marcos berges
```
Comment (astrojuanlu) [inline marcosberges.txt]: And, as a last step, can you remove this file? That way we don't pollute the root directory

File: names/marcosberges.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+name: marcos berges
```
Comment (astrojuanlu) [inline names/marcosberges.txt:1]: Perfect!

**Review by astrojuanlu — CHANGES_REQUESTED** [review]

File: marcosberges.txt (hunk 1 above)
Comment (marcosberges) [reply marcosberges.txt]: I thought It would delete here, since i deleted it locally. Could you tell me how to delete it?

Comment (astrojuanlu) [reply marcosberges.txt]: It's a bit counter intuitive, but after deleting it you have to `git add marcosberges.txt` (yes, `add` in this case tells git that it's been removed) and then commit the result.

Comment (marcosberges) [reply marcosberges.txt]: Done! Thanks for the help.

**Review by astrojuanlu — APPROVED** [review]

**Pull request merged**